uReplicator-Controller/src/main/java/com/uber/stream/kafka/mirrormaker/controller/core/OffsetMonitor.java

```diff
@@ -175,6 +172,19 @@ public void stop() throws InterruptedException {
     logger.info("OffsetMonitor closed");
   }
 
+  public boolean isHealthy() {
```

Comment:
Maybe change to isOffsetMonitorHealthy() in case we have more health check later.

Reply:
Since it's inside OffsetMonitor, isHealthy should be sufficient